jobs/delorean/jenkinsfiles/discovery/ga/github/Jenkinsfile

```diff
@@ -3,8 +3,8 @@
 
 def installationGitUrl = params.installationGitUrl ?: 'git@github.com:integr8ly/installation.git'
 def installationGitRef = params.installationGitRef ?: 'master'
-def githubToken = params.githubToken ?: 'jenkins-github-api-token'
-def githubCredentialsID = params.credentialId ?: 'jenkinsgithub'
+def githubCredentialsID = params.credentialId ?: 'githubjenkins'
+def githubSSHCredentialsID = params.sshCredentialId ?: 'jenkinsgithub'
```

Comment:
I assume none of these were ever coming in as params and we were always suing the default fallback variable?

Reply:
Yeah the default vars were always being used. We don't have this param set up in the jobs itself.